### ISSUE TITLE
Frontend Routes

### ISSUE THREAD

**Issue by linakherchi** [colineckert/everbrain#6]
- [x] Frontend routes contains wildcard variables written in `camelCase`
- [x] Correctly formatted
    - [x] Routes are displayed with `inline coding text` (backticks)

**Comment by linakherchi** [comment]
Thanks, Colin ! 

+ `DemoLoginForm` should be represented with a button in your `SessionForm` that will fill out input fields and logs in user on form submission - So no need to have a separate component for it.
+ What is the difference between `NotesIndex` and `NotesList` ?
+ ?tag=:tag_id - No need for a query string - you can immediately have `/:tagId`

**Comment by colineckert** [comment]
Thanks, Lina! Copy that on the `DemoLoginForm` and the `?tag=:tag_id` query string, not being necessary. I've updated my routes. 

My thinking is that the `NotesIndex` would house the whole notes sidebar, including the header with notebook title, selected tags if any, and a note count. Here's an example from my Evernote for what I think the `NotesIndex` will contain:

![Screen Shot 2021-02-05 at 3 27 04 PM](https://user-images.githubusercontent.com/60206665/107099528-cae31f80-67c6-11eb-88a7-67ee0757b23f.png)

The `NotesList` component will be rendered within the `NotesIndex` and contain the filtered list of notes and a `Link` for each note.

I changed the container name from `NotesSidebarContainer` to `NotesIndexContainer` for more clearly express that relationship.

Let me know if this doesn't make sense or you have any suggestions!
